gravitee-apim-console-webui/src/management/api/documentation-v4/components/documentation-new-page/documentation-new-page.component.ts

```diff
@@ -283,8 +284,8 @@ export class DocumentationNewPageComponent implements OnInit {
 
   private createPage(): Observable<Page> {
     const formValue = this.form.getRawValue();
-    // Only Markdown, Swagger, and AsyncAPI pages can be created
-    if (this.pageType !== 'MARKDOWN' && this.pageType !== 'SWAGGER' && this.pageType !== 'ASYNCAPI') {
+    // Only Markdown, Swagger, AsyncAPI, and AsciiDoc pages can be created
+    if (SUPPORTED_FOR_EDIT.includes(this.pageType) === false) {
```

Comment:
```suggestion
    if (!SUPPORTED_FOR_EDIT.includes(this.pageType)) {
```
Small simplification but not a deal-breaker